third_party/accessibility/ax/platform/ax_fragment_root_win_unittest.cc

```diff
@@ -94,7 +93,7 @@ TEST_F(AXFragmentRootTest, UIAFindItemByPropertyUniqueId) {
   {
     unique_id = AXPlatformNodeFromNode(GetRootAsAXNode())->GetUniqueId();
     unique_id_variant.Set(
-        SysAllocString(base::NumberToString16(-unique_id).c_str()));
+        SysAllocString((wchar_t*)base::NumberToString16(-unique_id).c_str()));
```

Comment:
Here and below: `static_cast`

Reply:
This had to be a `reinterpret_cast` rather than a `static_cast`, but I've updated it to use that.